cmd/ci-secret-bootstrap/README.md

```diff
@@ -13,22 +13,22 @@ in BitWarden and the targeting secret.
 secrets:
 - from:
     key-name-1:
-      bw-item: item-name-1
+      bwItem: item-name-1
```

Comment:
Can we keep the lower case with dashes? Pretty much all our YAML uses that format.

Reply:
Updated as discussed over slack.
We choose our tools' style over k8s' one.